docs/docs-beta/docs/dagster-plus/deployment/hybrid/agents/kubernetes.md

```diff
@@ -64,11 +68,32 @@ helm --namespace dagster-cloud upgrade agent \
     --values ./values.yaml
 ```
 
+## Troubleshooting tips
+
+You can see basic health information about your agent in the Dagster+ UI:
+
+[comment]: <> (TODO: Screenshot of Dagster+ Deployments agents tab)
+
+### View logs
+
+```shell
+kubectl --namespace dagster-cloud logs -l deployment=agent
+```
+
+
 ## Common configurations
 
-You can customize your Dagster+ agent using [Helm values](https://artifacthub.io/packages/helm/dagster-cloud/dagster-cloud-agent?modal=values). Some common configuration include
+There are three places to customize how Dagster interacts with Kubernetes:
+- *Globally* by configuring the Dagster+ agent using [Helm values](https://artifacthub.io/packages/helm/dagster-cloud/dagster-cloud-agent?modal=values)
+- *Per Project* by configuring the `dagster_cloud.yaml` file for your [code location](/dagster-plus/deployment/code-locations)
+- *Per Asset or Job* by adding tags to the [asset](/todo), [job](/todo), or [customizing the Kubernetes pipes invocation](/todo)
+
+Changes apply in a hierarchy, for example a customization for an asset will over-ride a default set globally in the agent configuration.
```

Comment:
s/over-ride/override

might be worth clarifying that it will only get overridden if it is specically set on the asset - like if you customize resources globally but then customize something else on the asset, the resources are still applied